Remotion/Web/Development.WebTesting.IntegrationTests/ScreenshotTest.cs

```diff
@@ -175,105 +173,6 @@ public void BadgeTest ()
-
-        builder.Crop(target.ForElementScopeScreenshot(), new WebPadding(30, 60, 30, 60));
-      };
-
-      var home = Start();
-      var mouseHelper = Helper.BrowserConfiguration.MouseHelper;
-      var element = home.Scope.FindId("tooltipTarget");
-
-      // ElementScope.Hover() cannot be used due to Chromedriver not moving the cursor to trigger the hover event.
-      mouseHelper.Hover(element);
-
-      // When drawing the Cursor tooltip, only ScreenshotTestingType.Desktop is supported.
-      Helper.RunScreenshotTestExact<ElementScope, ScreenshotTest>(element, ScreenshotTestingType.Desktop, test);
-
-      // Reset the cursor to the initial position (0, 0).
-      mouseHelper.Hover(Point.Empty);
-    }
-
-    /// <summary>
-    /// Tests that the <see cref="ScreenshotTooltipAnnotation"/> is annotated correctly.
-    /// </summary>
-    [Category("Screenshot")]
-    [Test]
-    public void DrawTooltip_WithFollowingBoxAnnotation ()
-    {
-      ScreenshotTestingDelegate<ElementScope> test = (builder, target) =>
-      {
-        var annotationTarget = Helper.BrowserConfiguration.BrowserAnnotateHelper.DrawTooltip(builder, target, GetTooltipStyleForCurrentBrowser());
-        builder.AnnotateBox(annotationTarget, Pens.Red);
-
-        builder.Crop(target.ForElementScopeScreenshot(), new WebPadding(30, 60, 30, 60));
-      };
-
-      var home = Start();
-      var element = home.Scope.FindId("tooltipTarget");
-      Helper.RunScreenshotTestExact<ElementScope, ScreenshotTest>(element, ScreenshotTestingType.Both, test);
-    }
-
-    /// <summary>
-    /// Tests that the <see cref="ScreenshotTooltipAnnotation"/> is annotated correctly.
-    /// </summary>
-    [Category("Screenshot")]
-    [Test]
-    public void DrawCursorTooltip_WithFollowingBoxAnnotation ()
-    {
-      ScreenshotTestingDelegate<ElementScope> test = (builder, target) =>
-      {
-        var annotationTarget = Helper.BrowserConfiguration.BrowserAnnotateHelper.DrawCursorTooltip(
-            builder,
-            Helper.MainBrowserSession,
-            "title",
-            GetTooltipStyleForCurrentBrowser());
-        builder.AnnotateBox(annotationTarget, Pens.Red);
-
-        builder.Crop(target.ForElementScopeScreenshot(), new WebPadding(30, 60, 30, 60));
-      };
-
-      var home = Start();
-      var mouseHelper = Helper.BrowserConfiguration.MouseHelper;
-      var element = home.Scope.FindId("tooltipTarget");
-
-      // ElementScope.Hover() cannot be used due to Chromedriver not moving the mouse to trigger the hover event.
-      mouseHelper.Hover(element);
-
-      Helper.RunScreenshotTestExact<ElementScope, ScreenshotTest>(element, ScreenshotTestingType.Both, test);
-
-      // Reset the cursor to the initial position (0, 0).
-      mouseHelper.Hover(Point.Empty);
-    }
-
     [Test]
     public void TooltipBounds_NotDrawnYet_ThrowsInvalidOperationException ()
```

Comment:
Depending on the outcome of the question of DrawTooltip, this test might also belong to UI-tests.